charts/kube-state-metrics/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- tariq1890
```

Comment:
@tariq1890 @mrueg are you still ok to maintain the chart?

Reply:
I'd like to keep maintaining it (I think @tariq1890 as well). We are open if others are interested as well.

Reply:
Yes, I'd still be interesting in maintaining the helm chart